Build failure: pkgsCross.riscv64.wipe

## Steps To Reproduce

Steps to reproduce the behavior:

1. build `nix build .#pkgsCross.riscv64.wipe`

## Build log

<!-- insert build log in code block in collapsable section -->

<details>

<summary>Build Log</summary>

```
wipe-riscv64-unknown-linux-gnu> riscv64-unknown-linux-gnu-gcc  *.o -o wipe
wipe-riscv64-unknown-linux-gnu> Running phase: installPhase
wipe-riscv64-unknown-linux-gnu> install flags: SHELL=/nix/store/5mh7kaj2fyv8mk4sfq1brwxgc02884wi-bash-5.2p37/bin/bash install
wipe-riscv64-unknown-linux-gnu> /nix/store/4s9rah4cwaxflicsk5cndnknqlk9n4p3-coreutils-9.5/bin/install -c -m 0755 -d /nix/store/3xdbhp6bnqbcflb22b4rs2s0z6gk4h21-wipe-riscv64-unknown-linux-gnu-2.3.1/bin
wipe-riscv64-unknown-linux-gnu> /nix/store/4s9rah4cwaxflicsk5cndnknqlk9n4p3-coreutils-9.5/bin/install -c -m 0755 -s ./wipe /nix/store/3xdbhp6bnqbcflb22b4rs2s0z6gk4h21-wipe-riscv64-unknown-linux-gnu-2.3.1/bin
wipe-riscv64-unknown-linux-gnu> /nix/store/4s9rah4cwaxflicsk5cndnknqlk9n4p3-coreutils-9.5/bin/install: cannot run 'strip': No such file or directory
wipe-riscv64-unknown-linux-gnu> /nix/store/4s9rah4cwaxflicsk5cndnknqlk9n4p3-coreutils-9.5/bin/install: strip process terminated abnormally
wipe-riscv64-unknown-linux-gnu> make: *** [Makefile:61: install] Error 1

```

</details>

## Additional context

Somehow the binary `strip` seems to be missing when cross compiling to RISCV

## Metadata

<!-- Please insert the output of running `nix-shell -p nix-info --run "nix-info -m"` below this line -->

```
 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.5, NixOS, 24.11 (Vicuna), 24.11.20241216.3945713`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.10`
 - channels(root): `"nixos-24.11"`
 - nixpkgs: `/nix/store/wj2qla569hnxwqfc26imv5hqbxc1rc27-source`
``` 

## Notify maintainers

<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->

@abbradar

---

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/wi/wipe/package.nix
+++ b/pkgs/by-name/wi/wipe/package.nix
@@ -32,7 +32,7 @@ stdenv.mkDerivation rec {
     description = "Secure file wiping utility";
     homepage = "https://wipe.sourceforge.net/";
     license = lib.licenses.gpl2Plus;
-    platforms = lib.platforms.all;
+    platforms = lib.platforms.unix;
     maintainers = [ lib.maintainers.abbradar ];
     mainProgram = "wipe";
   };

--- a/pkgs/by-name/wi/wipe/package.nix
+++ b/pkgs/by-name/wi/wipe/package.nix
@@ -14,6 +14,12 @@ stdenv.mkDerivation rec {
     sha256 = "180snqvh6k6il6prb19fncflf2jcvkihlb4w84sbndcv1wvicfa6";
   };
 
+  postPatch = ''
+    # Do not strip binary during install
+    substituteInPlace Makefile.in \
+      --replace-fail '$(INSTALL_BIN) -s' '$(INSTALL_BIN)'
+  '';
+
   nativeBuildInputs = [ autoreconfHook ];
 
   # fdatasync is undocumented on darwin with no header file which breaks the build.

--- a/pkgs/by-name/wi/wipe/package.nix
+++ b/pkgs/by-name/wi/wipe/package.nix
@@ -11,7 +11,7 @@ stdenv.mkDerivation rec {
 
   src = fetchurl {
     url = "mirror://sourceforge/wipe/${version}/${pname}-${version}.tar.bz2";
-    sha256 = "180snqvh6k6il6prb19fncflf2jcvkihlb4w84sbndcv1wvicfa6";
+    hash = "sha256-RjkWNw+bNbs0QZwsCuPcTApHHbMuhZWvodFMAze2GqA=";
   };
 
   postPatch = ''

--- a/pkgs/by-name/wi/wipe/package.nix
+++ b/pkgs/by-name/wi/wipe/package.nix
@@ -22,12 +22,12 @@ stdenv.mkDerivation rec {
 
   patches = [ ./fix-install.patch ];
 
-  meta = with lib; {
+  meta = {
     description = "Secure file wiping utility";
-    mainProgram = "wipe";
     homepage = "https://wipe.sourceforge.net/";
-    license = licenses.gpl2Plus;
-    platforms = platforms.all;
-    maintainers = [ maintainers.abbradar ];
+    license = lib.licenses.gpl2Plus;
+    platforms = lib.platforms.all;
+    maintainers = [ lib.maintainers.abbradar ];
+    mainProgram = "wipe";
   };
 }

